### DISCUSSION TITLE
Utterly confused about current status of BTCEUR support

### DISCUSSION THREAD

**Discussion by arminus** [ghostfolio/ghostfolio#3399] [Q&A]
Just started to evaluate self-hosted ghostfolio, main case would be to track crypto transactions with EUR as base currency with CSVs from cointracking being imported.

I researched for about an hour on open and closed tickets and discussions here, but I fail to understand the current status when it comes to being able to use symbols like BTC-EUR (or BTCEUR). Here https://github.com/ghostfolio/ghostfolio/discussions/2606#discussioncomment-7484118 it says that COINGECKO is now used (to get BTCEUR rates?), but trying to import a CSV with COINGECKO as DataSource and either BTC-EUR or BCTEUR as symbol shows a 404 for the CoinGeckoService - why?

Also, I can't add BTCEUR in Add Market Data - all I see there are USD symbols, apparently using Yahoo as source only.

All I managed was to enter a sample transaction manually, by using BTCUSD but unit and fee price as EUR - which it then on-the-fly converted to USD - but the displays it (mostly) in EUR elsewhere.

So this seems kind of "half-baked" in-progress and highly confusing for a newbie not trading in USD... Is this then basically https://buymeacoffee.com/ghostfolio/w/31930 ? Which still kind of begs the question why COINGECKO doesn't seem to work for me?

**Comment by dtslvr** [comment]
Hi @arminus

You can find the ticket about this topic here: https://github.com/ghostfolio/ghostfolio/issues/3320

I will close this discussion as a duplicate for better organization. Thanks for understanding!

**Reply by arminus** [reply]
Thanks for the fast reply - still confused about the role of coingecko though, which is supposedly used somehow according to other tickets I've seen, but which doesn't work for me.

**Reply by dtslvr** [reply]
_Coingecko_ is used as the underlying data provider. If you like to add an activity, search for _Bitcoin_ and select the entry with the ticker `bitcoin`.
